gax-java/gax/src/test/java/com/google/api/gax/batching/BatcherImplTest.java

```diff
@@ -830,7 +831,7 @@ public void testConstructors() throws InterruptedException {
     }
   }
 
-  @Test
+  @Test(timeout = 60000)
```

Comment:
This breaks an infinite loop, if any. The normal execution time is around 1 second. 1 minute should be enough.